services/src/test/scala/cromwell/pipeline/service/ServiceTestMode.scala

```diff
@@ -0,0 +1,7 @@
+package cromwell.pipeline.service
+
+trait ServiceTestMode
+
+case object Success extends ServiceTestMode
+case object WithException extends ServiceTestMode
```

Comment:
I like this idea, but let's do `case class WithException(exc: Exception)` to make it configurable

Reply:
That is really nice suggestion. There will be more exceptions in the next serviceImpl classes. Fixed.